docs/getting-started.qmd

```diff
@@ -14,33 +14,38 @@ The iRODS GUI works with any iRODS instance. Here we show how to configure the a
 ```sh
 git clone git@github.com:chStaiger/iBridges-Gui.git
 cd iBridges-Gui
-python3.10 -m pip install -r requirements.txt
+pip install -r requirements.txt
```

Comment:
I believe `python -m pip install` is the preferred invocation, but functionally either will work.  Also, in addition to `requirements.txt`, those developing/needing to test will need the packages in `requirements_dev.txt`.